Sources/Action/Action.swift

```diff
@@ -30,7 +34,11 @@ public final class Action<Input, Element> {
 
     /// Errors aggrevated from invocations of execute().
     /// Delivered on whatever scheduler they were sent from.
-    public let errors: Observable<ActionError>
+    public let errors: Observable<Error>
+    
+    /// Errors when Action cannot execute due to disabled.
```

Comment:
What does this actually means? "due to being disabled", you meant?

Reply:
Your action will be disabled by `enabledIf: Observable<Bool>` and `executing: Observable<Bool>`